plugins/doc_fragments/base.py

```diff
@@ -10,27 +10,31 @@
 class ModuleDocFragment(object):
     DOCUMENTATION = """
 options:
-  user:
-    description:
-      - The username to use for connecting to the Sensu API.
-        If this is not set the value of the SENSU_USER environment variable will be checked.
-    type: str
-    default: admin
-  password:
-    description:
-      - The Sensu user's password.
-        If this is not set the value of the SENSU_PASSWORD environment variable will be checked.
-    type: str
-    default: P@ssw0rd!
-  url:
-    description:
-      - Location of the Sensu backend API.
-        If this is not set the value of the SENSU_BACKEND_URL environment variable will be checked.
-    type: str
-    default: http://localhost:8080
-  namespace:
-    description:
-      - RBAC namespace to operate in.
-    type: str
-    default: default
+  auth:
+    description: Set of auth options
+    type: dict
+    suboptions:
+      user:
+        description:
+          - The username to use for connecting to the Sensu API.
+            If this is not set the value of the SENSU_USER environment variable will be checked.
+        type: str
+        default: admin
+      password:
+        description:
+          - The Sensu user's password.
+            If this is not set the value of the SENSU_PASSWORD environment variable will be checked.
+        type: str
+        default: P@ssw0rd!
+      url:
+        description:
+          - Location of the Sensu backend API.
+            If this is not set the value of the SENSU_BACKEND_URL environment variable will be checked.
+        type: str
+        default: http://localhost:8080
+      namespace:
+        description:
+          - RBAC namespace to operate in.
```

Comment:
Add
"If this is not set the value of the SENSU_NAMESPACE environment variable will be checked."